.gitignore

```diff
@@ -1,3 +1,2 @@
 venv
 .idea
-__pycache__
```

Comment:
We have you deleted this line

Reply:
I have already cleared it from the folder